{"name": "Class ServerException", "module": "java.rmi", "package": "java.rmi", "text": "A ServerException is thrown as a result of a remote method\n invocation when a RemoteException is thrown while processing\n the invocation on the server, either while unmarshalling the arguments or\n executing the remote method itself.\n\n A ServerException instance contains the original\n RemoteException that occurred as its cause.", "codes": ["public class ServerException\nextends RemoteException"], "fields": [], "methods": []}